Code.of.Conduct.DRiP.md

```diff
@@ -0,0 +1,61 @@
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment include:
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy toward other community members
+
+Examples of unacceptable behaviour include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or advances
+- Trolling, insulting/derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or electronic address, without explicit permission
+- Other conduct that could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers and community leaders are responsible for clarifying the standards of acceptable behaviour. They are expected to take appropriate and fair corrective action in response to any instances of unacceptable behaviour.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct or to ban temporarily or permanently any contributor for behaviours that they deem inappropriate, threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within and in public spaces when an individual represents the project or its community. Examples of representing a project or community include using an official project email address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behaviour may be reported by contacting the project team at [[your-email@example.com](mailto:your-email@example.com)] (or create a form as qiskit COC). All complaints will be reviewed and investigated, resulting in a necessary and appropriate response to the circumstances. The project team is obligated to maintain confidentiality concerning the reporter of an incident. Further details of specific enforcement policies may be posted separately.
+
+## Scientific Misconduct
+
+Collaboration members must perform research in a well-documented and ethically sound manner. Falsification of data or results, plagiarism, taking credit for other’s work or any other scientific misconduct will not be tolerated. “Data and research results should be recorded and maintained in a form that allows review, analysis, and reproduction by others.” Also, papers must include “a set of references that communicates the precedents, sources, and the context of the reported work. The deliberate omission of a pertinent author or reference is unacceptable.”2 
```

Comment:
What does the quoted text mean here?
Also, what is the number 2 at the end of the paragraph?
It seems like maybe you were wanting to reference the quotes? But I don't see what the citation it....